### PR TITLE
Fix display issues with goose wrapping and resolve a bug with translation of geese from left to right

### PR DESCRIPTION
**Pull request by harrisse**
This also changes the order of operation for applying hunger so that a goose can consist of one cell.

### CODE REVIEW

File: kaggle_environments/envs/hungry_geese/hungry_geese.js [hunk 1]
```diff
@@ -151,36 +151,53 @@ function renderer({
...
   };
 
-  const getRowCol = (cell) => [Math.floor(cell / columns), cell % columns];
+  // Javascript does not correctly handle mod of negatives
+  const mod = (number, divisor) => ((number % divisor) + divisor) % divisor;
+
+  const getRowCol = (cell) => [Math.floor(cell / columns), mod(cell, columns)];
+
+  const diff = (row, column, nextRow, nextColumn, rows, columns) => {
+    const north = mod(row - 1, rows) === nextRow
```
Comment (dster2) [inline kaggle_environments/envs/hungry_geese/hungry_geese.js:160]: nit: semicolons on these

File: kaggle_environments/envs/hungry_geese/hungry_geese.js [hunk 2]
```diff
@@ -151,36 +151,53 @@ function renderer({
...
   const geesePositions = {};
-  geese.forEach((goose, index) => {
+  geese.forEach((goose, gooseIndex) => {
     const active = goose.length > 0;
     let s = step;
     if (!active) {
       // Find the last step where the goose existed.
       while (s >= 0 && !goose.length) {
-        goose = environment.steps[--s][0].observation.geese[index];
+        goose = environment.steps[--s][0].observation.geese[gooseIndex];
```
Comment (dster2) [inline kaggle_environments/envs/hungry_geese/hungry_geese.js:177]: With `--s` wouldn't you be able to get `-1` if `s == 0`?  Or is that intended?

File: kaggle_environments/envs/hungry_geese/hungry_geese.js [hunk 3]
```diff
@@ -151,36 +151,53 @@ function renderer({
...
 
   // Organize the geese positions for rendering.
   const geesePositions = {};
-  geese.forEach((goose, index) => {
+  geese.forEach((goose, gooseIndex) => {
     const active = goose.length > 0;
     let s = step;
     if (!active) {
       // Find the last step where the goose existed.
       while (s >= 0 && !goose.length) {
```
Comment (dster2) [inline kaggle_environments/envs/hungry_geese/hungry_geese.js:176]: nit: Perhaps negligible, but I think this could be binary search instead?

**Review by dster2 — APPROVED** [review]
Thanks Sam!

File: kaggle_environments/envs/hungry_geese/hungry_geese.js (hunk 3 above)
Comment (harrisse) [reply kaggle_environments/envs/hungry_geese/hungry_geese.js:176]: Fair point, but negligible utility in this case as it runs in browser and there are very few elements to traverse.

**Pull request merged**